components/Common/Search/utils.ts

```diff
@@ -1,7 +1,18 @@
+import type { Result } from '@orama/orama';
```

Comment:
IMHO this should be put on `utils/search`.

Reply:
Moved it